task_1/src/main/java/task/Reversed.java

```diff
@@ -0,0 +1,60 @@
+package task;
+
+import java.util.Scanner;
+
+public class Reversed {
+
+        public static void main(String[] args) {
+            System.out.println("Enter to string\n");
+            Scanner scr = new Scanner(System.in);
+            String s = scr.nextLine();
+
+            String[] input = s.split(" ");
+            for (int i = 0; i < input.length; i++) {
+                input[i] = getReversedString(input[i]);
+            }
+            s = buildResult(input);
+            System.out.println();
+            System.out.println("String after revers\n\n" + s);
+        }
+
+    public static String getReversedString(String input) {
```

Comment:
String input also should be renamed to word

Reply:
Done